modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/share/ShareListCreateViewModel.kt

```diff
@@ -39,7 +39,7 @@ class ShareListCreateViewModel @Inject constructor(
     init {
         viewModelScope.launch {
             val podcasts = podcastManager.findPodcastsOrderByTitle()
-            mutableState.value = mutableState.value.copy(podcasts = podcasts)
+            mutableState.value = mutableState.value.copy(podcasts = podcasts.filter { it.canShare })
```

Comment:
We decided to exclude private ones from this list. See: pdeCcb-7Yf-p2#comment-6356